### PR TITLE
chore(tracing): Cleanup

### PR DESCRIPTION
**Pull request by jacobmarble**
_Briefly describe your proposed changes:_

Various tracing cleanups:
- Add parent spans where child spans were triggering root spans (eg Scheduler, Flux from/to)
- Remove a Tracer pointer (we use `opentracing.GlobalTracer`)
- Add some `tracing.LogError` love
- Inject and extract HTTP headers with library rather than copy-pasted code
- Log file/line from `tracing.LogError`
- Add `tracing.StartSpanFromContextWithOperationName`
- Generally replace opentracing usage with `kit/tracing` for consistency and better spans

  - [x] Rebased/mergeable
  - [ ] Tests pass

### CODE REVIEW

File: query/stdlib/influxdata/influxdb/from.go [hunk 1]
```diff
@@ -649,6 +651,9 @@ func (FromKeysRule) Rewrite(keysNode plan.Node) (plan.Node, bool, error) {
 }
 
 func createFromSource(prSpec plan.ProcedureSpec, dsid execute.DatasetID, a execute.Administration) (execute.Source, error) {
+	span, ctx := tracing.StartSpanFromContext(context.TODO())
```
Comment (docmerlin) [inline query/stdlib/influxdata/influxdb/from.go]: Are you sure about this being a TODO? it seems that context.Background() would work better here.

File: query/stdlib/influxdata/influxdb/source.go [hunk 2]
```diff
@@ -114,6 +115,9 @@ func (s *readFilterSource) run(ctx context.Context) error {
 }
 
 func createReadFilterSource(s plan.ProcedureSpec, id execute.DatasetID, a execute.Administration) (execute.Source, error) {
+	span, ctx := tracing.StartSpanFromContext(context.TODO())
```
Comment (docmerlin) [inline query/stdlib/influxdata/influxdb/source.go]: Again, TODO -> Background

File: query/stdlib/influxdata/influxdb/to.go [hunk 3]
```diff
@@ -439,6 +440,9 @@ func (s Stats) Update(o Stats) {
 }
 
 func writeTable(t *ToTransformation, tbl flux.Table) error {
+	span, ctx := tracing.StartSpanFromContext(context.TODO())
```
Comment (docmerlin) [inline query/stdlib/influxdata/influxdb/to.go]: I'm not sure about this one.  Are you intending to eventually modify writeTable to accept a context?

File: query/stdlib/influxdata/influxdb/from.go (hunk 1 above)
Comment (jacobmarble) [reply query/stdlib/influxdata/influxdb/from.go]: Just a note for the query team. I figure they're neck-deep in refactors, so this is enough of a billboard.

File: query/stdlib/influxdata/influxdb/to.go (hunk 3 above)
Comment (jacobmarble) [reply query/stdlib/influxdata/influxdb/to.go]: That would be the idea, yes. Again, query team's decision. When you touch I/O, it's a good place for tracing and a good place for context.

**Pull request merged**